gradle-plugin/build.gradle

```diff
@@ -0,0 +1,20 @@
+plugins {
+    id 'org.jetbrains.kotlin.jvm' version '1.3.21'
```

Comment:
I recommend to declare a plugin dependency in the root script:
```
// <root>/build.gradle
plugins {
    id 'org.jetbrains.kotlin.jvm' version '1.3.21' apply false
}
```
Then the plugin could be applied in any subproject like this:
```
plugins {
    id 'org.jetbrains.kotlin.jvm' 
}
```
The reasons for declaring a plugin dependency in root project are:
1. To avoid repeating a version.
2. To avoid repeated classloading. Gradle creates a new classloader per subproject, plugin's classes are loaded in the first classloader from the root. E.g. for the following subprojects hierarchy `:a:b`, `:a:c` Gradle would create three classloaders `classloader a <- classloader b`, `classloader a <- classloader c` (where `a <- b` is "a is a parent of b").
If root project does not declare a dependency on the plugin, then different plugin instances would be loaded in `b` and `c` classloaders, which leads to slightly worse performance and, in case of Kotlin plugin, subtle bugs.